website/docs/contributing/writing-guide.md

```diff
@@ -46,10 +46,17 @@ Use sentence case for headers: `### Some header with URL`, not `### Some Header
 Please use markdown file paths for links, not URL paths,
 because it works for both editors/IDEs (Ctrl/⌘+click works) and Docusaurus.
 Always add `.md` extension to the file paths.
+Use relative paths for links to files in the same directory or in a parent directory.
```

Comment:
Does that imply that for files in subdirectories, absolute links should be used?